_playground/blueprint.json

```diff
@@ -0,0 +1,31 @@
+{
+	"landingPage": "/wp-admin/site-editor.php",
+	"features": {
+		"networking": true
+	},
+	"steps": [
+        {
+			"step": "login"
+		},
+		{
+			"step": "installPlugin",
+			"pluginZipFile": {
+				"resource": "wordpress.org/plugins",
+				"slug": "gutenberg"
+			},
+			"options": {
+				"activate": true
+			}
+		},
```

Comment:
Do we need GB? (I'm fine with it, just genuinely curious 🙂)

Reply:
All set!